studio/http_artifact_store.py

```diff
@@ -0,0 +1,71 @@
+import logging
```

Comment:
Is this purely for the API server or are the runners supposed to support this protocol ?

Reply:
That's for the API client rather; but if the runner has direct access to the artifacts storage (i.e. if it can read/write from storage using qualified location) this logic is not necessary